### PR TITLE
[expanse-272] Add additional checks to test playbook and update input values

### PR DESCRIPTION
**Pull request by andrew-expanse**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27279

## Description
This PR updates the test playbook input to replace a domain that was removed from our test environment and it also adds some conditional checks to ensure that the playbook exits if an input is removed in the future. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [x] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by andrew-expanse** [comment]
@trorabaugh and I will review this together on 8/11/20.

**Comment by andrew-expanse** [comment]
@trorabaugh and I did a live demo walkthrough of the changes to the test playbook and everything should be fixed and less prone to break this way in the future.

**Comment by trorabaugh** [comment]
Let us know if you need anything else on this.

**Comment by trorabaugh** [comment]
@andrew-expanse you may need to review the circle ci and potentially update any merge conflicts.

File: Packs/Expanse/ReleaseNotes/1_1_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Expanse
+- Updated the author field to reflect Expanse contact email
```
Comment (barchen1) [inline Packs/Expanse/ReleaseNotes/1_1_1.md:4]: No need to change the author, it should stay "Expanse"

File: Packs/Expanse/pack_metadata.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
     "name": "Expanse",
     "description": "The Expanse App for Demisto leverages the Expander API to retrieve network exposures and create incidents in Demisto.",
     "support": "partner",
-    "currentVersion": "1.1.0",
-    "author": "Expanse",
+    "currentVersion": "1.1.1",
+    "author": "help@expanseinc.com",
```
[the remark this answers is not in the document]
Comment (andrew-expanse) [reply Packs/Expanse/pack_metadata.json]: Sorry, must've been some miscommunication earlier - I've removed this now.

**Review by barchen1 — CHANGES_REQUESTED** [review]
Packs/AutoFocus/Integrations/FeedAutofocus/demo_video/AutoFocus_Feed_demo.mp4 - This file looks unrelated to this PR.

**Comment by andrew-expanse** [comment]
> Packs/AutoFocus/Integrations/FeedAutofocus/demo_video/AutoFocus_Feed_demo.mp4 - This file looks unrelated to this PR.

ughh I have no idea how this got here - I'll try to remove it.

**Review by barchen1 — APPROVED** [review]

**Comment by barchen1** [comment]
@andrew-expanse Hi Andrew 
I tried to to run the `test-Expanse` test playbook and it failed on `domain` task.
The error I get when I tried to run `!domain domain=shack.com` is: `'NoneType' object has no attribute 'get'`
We need to fix this command and make sure the testplaybook pass.

Screen Shot 2020-10-07 at 13 47 07

Screen Shot 2020-10-07 at 13 47 37

**Comment by andrew-expanse** [comment]
Hi @barchen1 I'm looking into this now. It appears it may be an API related issue on our end. I've contacted our product engineering team and will update ASAP once I know more about why this change occurred and whether it was intentional.

**Pull request merged**